packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -333,3 +334,101 @@ export const ReadOnly: StoryObj<AutocompleteType> = {
     value: { label: "Tycho Station" },
   },
 };
+
+type MoonMeta = {
+  id: string;
+  label: string;
+  diameterInKm: number;
+  description: string;
+};
+
+type JupiterMoonsAutocomplete = typeof Autocomplete<MoonMeta, boolean, boolean>;
+
+const jupiterGalileanMoons: MoonMeta[] = [
+  {
```

Comment:
I'd like to meet to go over the Storybook implementation, so I can better understand it.

Reply:
Sure. DM me when you want to go over it.

Reply:
We can do this when I'm back. Storybook stuff can "break" between v1 versions. It's the component itself that's problematic.